mne/tests/test_cov.py

```diff
@@ -75,6 +76,24 @@ def test_compute_whitener(proj, pca):
         assert pca is False
         assert_allclose(round_trip, np.eye(n_channels), atol=0.05)
 
+    raw.info['bads'] = [raw.ch_names[0]]
+    picks = pick_types(raw.info, meg=True, eeg=True, exclude=[])
+    with pytest.warns(RuntimeWarning, match='Too few samples'):
+        cov2 = compute_raw_covariance(raw, picks=picks)
+        cov3 = compute_raw_covariance(raw, picks=None)
+    assert (cov2['data'][1:, 1:] == cov3['data']).all()
```

Comment:
It is better to use `assert_array_equal` here, it will give a more informative error
```suggestion
    assert_array_equal(cov2['data'][1:, 1:], cov3['data'])
```